Code/src/main/java/nl/utwente/viskell/ui/components/Pair.java

```diff
@@ -0,0 +1,16 @@
+package nl.utwente.viskell.ui.components;
+
+/**
+ * A generic 2-tuple
+ * @param <A> type of the first element, a
+ * @param <B> type of the second element, b
+ */
+public class Pair<A, B> {
```

Comment:
That Pair has the same semantics as Entry: a key-value pair. This is not a key-value pair, but a pair of values.
I agree that components isn't the greatest place for it to reside.